### PR TITLE
added an example

### PR DESCRIPTION
**Pull request by julienkessels**

### CODE REVIEW

File: src/software/ros/basics/example.md
Comment (azerupi) [inline src/software/ros/basics/example.md]: Did you try your example? If you had, you would have known that the increment operator doesn't exist in python.. 😉

**Review by azerupi — CHANGES_REQUESTED** [review]
I can't merge this. The code is not working.
Before I merge this, you need to make sure the code works and correct the mistakes.

**Comment by julienkessels** [comment]
should be fine now

**Review by azerupi — APPROVED** [review]

**Pull request merged**
